File: Data Structures & Algorithms/majority-element/submission-1.cpp
class Solution {
public:
    int majorityElement(vector<int>& nums) {
    int ele=nums[0];
    int count=0;

    for(int i=0;i<nums.size();i++){
        if(nums[i]==ele){
            count++;
        }
        else{
            count--;
        }
        if(count==0){
            ele=nums[i+1];
        }
    }
    if(count!=0){
        return ele;
    }
    return 0;
    }
};

File: Data Structures & Algorithms/majority-element-ii/submission-1.py
class Solution:
    def majorityElement(self, nums: List[int]) -> List[int]:
        c=Counter(nums)

        li=c.most_common()
        res=[]
        for i in range(len(li)):
            if li[i][1]>math.floor(len(nums)/3):
                res.append(li[i][0])

        return res